client/packages/reports/src/components/ReportWidget.tsx

```diff
@@ -142,13 +142,13 @@ export const ReportWidget: React.FC<PropsWithChildren<ReportWidgetProps>> = ({
                         />
                       </Grid>
                     </Link>
-                    <ReportArgumentsModal
-                      report={reportWithArgs}
-                      onReset={() => setReportWithArgs(undefined)}
-                      onArgumentsSelected={reportArgs}
-                    />
                   </React.Fragment>
                 ))}
+                <ReportArgumentsModal
```

Comment:
doesn't fix it for me 😢 
Looking at the source - there are three modals showing:


![image](https://github.com/user-attachments/assets/22fd5b7b-55cb-457e-bb84-26d23ee98fbf)